### PR TITLE
Synonyms work

### PR DESCRIPTION
**Pull request by fredkingham**

### CODE REVIEW

**Comment by fredkingham** [comment]
fixes #6

**Comment by fredkingham** [comment]
@davidmiller merge conflicts fixed

File: README.md [hunk 1]
```diff
@@ -165,3 +165,7 @@ A details json field on the element, allows the capture of miscellanious metadat
 ### Metadata
 
 This brings in the field result template for each test.
+
+### Synonyms
+
+If a class has synonyms, for example a Chest Xray is sometimes known as CX, these should be declared in _synonyms as part of the class. These will be displayed in the lab test input and will return the same form as the actual model.
```
Comment (davidmiller) [inline README.md:239]: A code example might make this really clear...

**Comment by davidmiller** [comment]
Looks great !

**Pull request merged**